Enrich the Creating an Access Control List (ACL) rule doc (#6946)
magento/devdocs: src/guides/v2.3/ext-best-practices/tutorials/create-access-control-list-rule.md

     <add id="Vendor_MyModule::menu" title="Custom Menu" module="Vendor_MyModule" sortOrder="10" resource="Vendor_MyModule::menu"/>
     <add id="Vendor_MyModule::create" title="Create" module="Vendor_MyModule" sortOrder="10" parent="Vendor_MyModule::menu" action="custommenu/create/index" resource="Vendor_MyModule::create"/>
     <add id="Vendor_MyModule::delete" title="Delete" module="Vendor_MyModule" sortOrder="20" parent="Vendor_MyModule::menu" action="custommenu/delete/index" resource="Vendor_MyModule::delete"/>
     <add id="Vendor_MyModule::view" title="View" module="Vendor_MyModule" sortOrder="30" parent="Vendor_MyModule::menu" action="custommenu/view/index" resource="Vendor_MyModule::view"/>
    </menu>
</config>
```

| Attribute | Description |
| --------- | ----------- |
| `id` | Unique string. Should be in the format: `Vendor_ModuleName::resourceName` |
| `title` | Title which is displayed in menu bar|
| `module` | Module which containing the current menu |
| `sortOrder` | Position in which menu to be displayed |
| `parent` | The another menu which is parent of current menu |

> | `action` | Url of the page which needs to be display after click the menu. It should be in following format: `front_name/controller_path/action` |

Reviewer: ```suggestion
| `action` | Url of the page which needs to be displayed after clicking the menu. It should be in following format: `front_name/controller_path/action` |
```

If the user doesn't have permission, the action page displays an "Access Denied" message.

### Page content partial restriction for admin users

With ACL it is also possible to [render layout blocks dynamically]({{ page.baseurl }}/frontend-dev-guide/layouts/xml-manage.html#ref_config_block) on the page.

It is enough to set the block's value for `aclResource` attribute:

```xml
<block class="Vendor\MyModule\Block\Adminhtml\Type" name="block.example" aclResource="Vendor_MyModule::view_additional">
    <!-- ... -->
</block>
```

> The `view/adminhtml/layout/custommenu_view_index.xml` example file below contains two blocks that shows information to the end user. One of which is accessible only to users with ACL `Vendor_MyModule::view_additional` allowed.

Reviewer: ```suggestion
The `view/adminhtml/layout/custommenu_view_index.xml` example file below contains two blocks that show information to the end-user. One of which is accessible only to users with ACL `Vendor_MyModule::view_additional` allowed.
```

If the user doesn't have permission, the action page displays an "Access Denied" message.

### Page content partial restriction for admin users

With ACL it is also possible to [render layout blocks dynamically]({{ page.baseurl }}/frontend-dev-guide/layouts/xml-manage.html#ref_config_block) on the page.

It is enough to set the block's value for `aclResource` attribute:

```xml

> <block class="Vendor\MyModule\Block\Type" name="block.example" aclResource="Vendor_MyModule::view_additional">

Reviewer: Just a minor edit if we are talking about admin panel 

```suggestion
<block class="Vendor\MyModule\Block\Adminhtml\Type" name="block.example" aclResource="Vendor_MyModule::view_additional">
```
